doc/source/examples/incompressible-flow/2d-sudden-expansion-flow/2d-sudden-expansion-flow.rst

```diff
@@ -0,0 +1,332 @@
+====================================
+Sudden-Expansion Flow
+====================================
+
+--------
+Features
+--------
+
+- Solver: ``lethe-fluid`` (with Q1-Q1)
+- Steady-state and transient solutions
+- Comparison with experimental and numerical results
+
+----------------------------
+Files Used in This Example
+----------------------------
+
+All files mentioned below are located in the example's folder (``examples/incompressible-flow/2d-sudden-expansion-flow``).
+
+- Geometry file: ``two-dimensional-jet.geo``
+- Mesh file: ``two-dimensional-jet.msh``
+- Parameter file for steady-state case: ``Reynolds70-jet.prm``
+- Parameter file for transient case: ``Reynolds610-jet.prm``
+- Postprocessing Python script for computing velocity distribution: ``velocity_profiles.py``
+- Reference data for velocity profile comparison: ``.txt`` files in ``Reference-Data`` folders
+
+-----------------------
+Description of the Case
+-----------------------
+
+A bidimensional flow goes past a symmetric sudden expansion. The flow enters from the left inlet at a velocity :math:`\mathbf{u} = [1, 0, 0]`, where the inlet slot height is :math:`h = 10mm`. At :math:`x = L_{inlet} = 8h` the flow is fully developed, and a sudden expansion with ratio 2 is placed (:math:`d = h/2`). The outlet length is given by :math:`L_{outlet} = 100h`; all dimensions are identified in the figure bellow.
+
+.. image:: image/expansion-geometry.png
+
+The sudden-expansion flow is a common fluid dynamics problem in which a fluid jet discharge can be either confined by boundaries or not (free jet). Here a symmetric boundary setup is simulated; at low Reynolds number, it has been reported that the steady-state flow remains symmetric. As the Reynolds number increases, however, the flow symmetry is lost and asymmetric recirculation zones appear. 
+Both stedy-state (:math:`\textrm{Re} =70`) and transient case (:math:`\textrm{Re} = 610`) are presented here.
```

Comment:
```suggestion
Both steady-state (:math:`\textrm{Re} =70`) and transient case (:math:`\textrm{Re} = 610`) are presented here.
```
You talked about symmetric and non-symmetric before. Could be a good idea to say that the steady case is the symmetric and the unsteady one is the non-symmetric :)